src/main/java/opendota/Parse.java

```diff
@@ -307,6 +314,8 @@ else if (cle.getType() == DOTA_COMBATLOG_TYPES.DOTA_COMBATLOG_GOLD) {
             else if (cle.getType() == DOTA_COMBATLOG_TYPES.DOTA_COMBATLOG_XP) {
                 combatLogEntry.xp_reason = cle.getXpReason();
             }
+            combatLogVisitors.visit(combatLogEntry);
```

Comment:
idk, we'll loose the polymorphic behavior of visitors if we always set the values on parser and CombatLogVisitors will be useless.